src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -40,11 +44,29 @@ public static Person[] getSamplePersons() {
         };
     }
 
+    public static Booking[] getSampleBookings() {
+        return new Booking[] {
+                new Booking(new Description("David's Party"),
```

Comment:
this might be kinda weird, because our app is suppose to be targeting classes and consults, but i guess the functionality is so generic that any booking works

Reply:
+1 on this! Maybe change it to another consult!